gorder.go

```diff
@@ -1,29 +1,24 @@
 package gorder
+import "errors"
 
-import (
-	"errors"
-	"regexp"
-)
+type Algo int
```

Comment:
It might be more readable if we used a non-abbreviated name for an exported type. What do you think?

```suggestion
type Algorithm int
```